src/test/test-install-root.c

```diff
@@ -1043,6 +1043,151 @@ static void test_preset_multiple_instances(const char *root) {
         unit_file_changes_free(changes, n_changes);
 }
 
+static void verify_one(
+                const UnitFileInstallInfo *i,
+                const char *alias,
+                int expected,
+                const char *updated_name) {
+        int r;
+        static const UnitFileInstallInfo *last_info = NULL;
+        _cleanup_free_ char *alias2 = NULL;
+
+        if (i != last_info)
+                log_info("-- %s --", (last_info = i)->name);
```

Comment:
no a fan of such cryptic code

Reply:
It's a test function ;) It's long enough already, and this allows the boilerplate code to be concise and not obscure the actual tests that much.